packages/pouch-storage/src/PouchStorageTable.ts

```diff
@@ -138,22 +140,36 @@ export class PouchStorageTable<T extends StorageItem = StorageItem>
 
   private currentViewportData?: ViewportData<T>;
 
-  constructor(databaseName: string) {
-    this.db = new PouchDB<T & PouchStorageItem>(`${DB_PREFIX}${databaseName}`);
+  constructor(
+    databaseName: string,
+    dbOptions?: PouchDB.HttpAdapter.HttpAdapterConfiguration
```

Comment:
This isn't the correct overload that we're using - guessing you Ctrl+Click'ed through to the definition, and came across this constructor with `HttpAdapterConfiguration`, which is the last overload defined because it is referenced last in the [@types/pouchdb-browser](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/62061c318fa41cc80b099cabb6f8a5c8552da873/types/pouchdb-browser/index.d.ts#L10).

We don't use `adapter: 'http'` though, we're just using the default adapter ([IndexedDB in the browser](https://pouchdb.com/adapters.html)), which is the default overload (which in TS for some reason appears as 4/4 if you type `new PouchDB(...)` ... I wonder why it shows them in the reverse order of declaration).
![image](https://user-images.githubusercontent.com/4505624/225332788-9714c9e9-7373-4df5-8a16-b1a977c45210.png)

In any case, this should just use `PouchDB.Configuration.DatabaseConfiguration`, where adapter isn't defined.
```suggestion
    dbOptions?: PouchDB.Configuration.DatabaseConfiguration
```